### PR TITLE
Unify bundler_setup guide into version-independent one

### PR DESCRIPTION
**Pull request by tnir**
### What was the end-user problem that led to this PR?

```diff
$ diff -u source/{v1.15/,}guides/bundler_setup.html.haml 
--- source/v1.15/guides/bundler_setup.html.haml 2022-07-26 07:53:54.000000000 +0000
+++ source/guides/bundler_setup.html.haml       2022-07-26 07:53:54.000000000 +0000
@@ -1,8 +1,8 @@
 ---
-title: Using Bundler with plain Ruby
+title: How to use Bundler with Ruby
 ---
 .container.guide
-  %h2 Bundler.setup
+  %h2 How to use Bundler with Ruby
 
   .contents
     .bullet
@@ -59,9 +59,9 @@
 
     .bullet
       .description
-        This will automatically discover your <code>Gemfile</code>, and make all of the gems in
+        This will automatically discover your <code>Gemfile</code> and make all of the gems in
         your <code>Gemfile</code> available to Ruby (in technical terms, it puts the gems "on the
-        load path"). You can think of it as an adding some extra powers to <code>require
+        load path"). You can think of it as adding some extra powers to <code>require
         'rubygems'</code>.
 
     .bullet
@@ -92,7 +92,7 @@
         source 'https://rubygems.org'
 
         gem 'rails', '5.0.0'
-        gem 'rack-cache', :require => 'rack/cache'
+        gem 'rack-cache', require: 'rack/cache'
         gem 'nokogiri', '~> 1.4.2'
     .bullet
       .description
```

### What was your diagnosis of the problem?

It is possible to remove old contents without any migration.

### What is your fix for the problem, implemented in this PR?

Remove duplication.

### Why did you choose this fix out of the possible options?

`https://bundler.io/v1.15/guides/bundler_setup.html` will be removed (with redirection).

Signed-off-by: Takuya Noguchi [takninnovationresearch@gmail.com](https://github.com/sponsors/tnir)

### CODE REVIEW

**Review by deivid-rodriguez — APPROVED** [review]

**Review by tnir — COMMENTED** [review]
Oops, redirection configuration is wrong... 😩

**Review by tnir — COMMENTED** [review]
Now it works well on every redirection 🎉

File: config.rb [hunk 1]
```diff
@@ -110,10 +110,15 @@
...
 
-  %w[bundler_setup bundler_sharing deploying faq git_bisect git groups rails sinatra updating_gems].each do |filename|
+  %w[bundler_sharing deploying faq git_bisect git groups rails sinatra updating_gems].each do |filename|
     redirect "v#{version}/#{filename}.html", to: "v1.15/guides/#{filename}.html"
   end
 
+  # Redirect versioned-guides (which are not localizable) on v1.15 and below to version-independent guides
+  %w[bundler_setup].each do |filename|
+    redirect "v#{version}/#{filename}.html", to: "guides/#{filename}.html"
+  end
```
Comment (deivid-rodriguez) [inline config.rb:120]: Needs to go above the `next if version == "1.15"` line now?

Comment (tnir) [reply config.rb:120]: No. Check out my previous work #669 and #668. Also @indirect's initial work: #322 on v1.15.

**Pull request merged**